installer/templates/phx_assets/package.json

```diff
@@ -17,9 +17,9 @@
     "copy-webpack-plugin": "^4.5.0",
     "css-loader": "^2.1.1",
     "mini-css-extract-plugin": "^0.4.0",
-    "optimize-css-assets-webpack-plugin": "^4.0.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
     "terser-webpack-plugin": "^1.1.0",
     "webpack": "4.4.0",
```

Comment:
Just asking, because I've stumbled upon `npm audit` security alerts too: shouldn't this version be changed to `^4.23.1`, like [here](https://github.com/suryart/phoenix/blob/94331443a5190c38c515b83ee54f3cb83c4a93ca/assets/package.json#L25)?